tests/perf-test/SearchPage.perf-test.js

```diff
@@ -156,10 +152,6 @@ test.skip('[Search Page] should search in options list', async () => {
         fireEvent.changeText(input, mockedPersonalDetails['88'].login);
         await act(triggerTransitionEnd);
         await screen.findByText(mockedPersonalDetails['88'].login);
-
-        fireEvent.changeText(input, mockedPersonalDetails['45'].login);
-        await act(triggerTransitionEnd);
-        await screen.findByText(mockedPersonalDetails['45'].login);
```

Comment:
Why is this removed just for my knowledge, was the double option selection flaky?

Reply:
removed it just to optimise the time of test execution, previously we modified input twice, but we decided that it is enough to do it once - the performance still will be measured, but test execution time will drop